guides/common/modules/ref_accessing-ansible-modules-docs.adoc

```diff
@@ -0,0 +1,25 @@
+[id="accessing-ansible-modules-docs_{context}"]
+== Accessing {Project} Ansible Modules Documentation
```

Comment:
Why do users need to read this procedure? It repeats the same information that was said above, even the commands. The link to theforeman.org can be found in the Ansible Galaxy linked in the listing section. I think you can simply remove this file.

Reply:
Removed.

Reply:
Please remove the file too.